deploy/acm-policies/policy-dedicated-admin.yaml

```diff
@@ -0,0 +1,1019 @@
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+    annotations:
+        policy.open-cluster-management.io/categories: CM Configuration Management
+        policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+        policy.open-cluster-management.io/standards: NIST SP 800-53
+    name: dedicated-admin-rbac
+    namespace: openshift-rbac-policies
+spec:
+    disabled: false
+    policy-templates:
+        - objectDefinition:
+            apiVersion: policy.open-cluster-management.io/v1
+            kind: ConfigurationPolicy
+            metadata:
+                name: dedicated-admin-rbac
+            spec:
+                object-templates:
+                    - complianceType: musthave
+                      objectDefinition:
+                        aggregationRule:
+                            clusterRoleSelectors:
+                                - matchExpressions:
+                                    - key: managed.openshift.io/aggregate-to-dedicated-admins
+                                      operator: In
+                                      values:
+                                        - cluster
+                                - matchExpressions:
+                                    - key: olm.opgroup.permissions/aggregate-to-admin
+                                      operator: In
+                                      values:
+                                        - openshift-logging
+                                        - openshift-operators
+                                        - openshift-operators-redhat
+                                - matchExpressions:
+                                    - key: olm.owner.kind
+                                      operator: In
+                                      values:
+                                        - OperatorGroup
+                                    - key: olm.owner
+                                      operator: NotIn
+                                      values:
+                                        - openshift-cluster-monitoring
+                                        - olm-operators
+                                        - cloud-ingress-operator
```

Comment:
cloud-ingress-operator isn't present on hypershift clusters, so this could likely be removed. Same for some of the other values on this list and the namespace list.

Reply:
I have removed cloud-ingress-operator,openshift-velero, rbac-permission-operator from this list and namespace list.